FirebaseCore/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v6.7.1 -- M71
+- [fixed] Fixed `FirebaseApp`s `bundleID` verification, allowing exact `bundleID` matches
+  for extensions. (#5531)
```

Comment:
Thank you for updating the log. Could you please put the issue number here #5126

Reply:
Nice catch, thanks!